Resources/Prototypes/Entities/Objects/Weapons/Throwable/clusterbang.yml

```
@@ -323,3 +323,38 @@
   - type: ContainerContainer
     containers:
       cluster-payload: !type:Container
+
+
+
+- type: entity
+  parent: GrenadeBase
+  id: ClusterBangCleanade
+  name: cluster cleanade
+  description: Launches three cleanade. when it too messy for one cleanade how about 3 cleanade.(janitor.co does not responsible any mass slipping accident with this products)
```

```suggestion
  description: Launches three cleanades after the timer runs out.
```
this is parity with the clusterbang description